scripts/sct_apply_transfo.py

```diff
@@ -299,7 +299,8 @@ def apply(self):
 
         if islabel:
             sct.printv("\nTake the center of mass of each registered dilated labels...")
-            cubic_to_point(im_src_reg).save(path=fname_out)
+            labeled_img = cubic_to_point(im_src_reg)
+            labeled_img.save(path=fname_out)
```

Comment:
Ah! This is present in more places than just `cubic_to_point`. My apologies, I didn't point all of them out in my initial review. 

But, it's easy enough to spot, e.g. `").save"` can be searched for.

Reply:
Is this bad though? If the returned image won't be used it's fair to no store it in a variable right?

Reply:
Ah, right. This is the same tradeoff as the "make a copy for RPI conversion", isn't it? (readability vs. memory usage)

I tend to err on the side of readability, but you raise a really good point. Honestly, I'm not knowledgable enough about Python's garbage collection, so it's hard for me to say how much of a cost it really is.

For now I'm happy to go with whatever your preference is. :)

Reply:
> Ah, right. This is the same tradeoff as the "make a copy for RPI conversion", isn't it? (readability vs. memory usage)

In this specific case no, it is simply [method chaining](https://en.wikipedia.org/wiki/Method_chaining) for the sake of readability. `cubic_to_point()` returns an `Image,` but we only save it and do nothing else with it so why store it in a variable.

```python
cubic_to_point(img).save()
```
is equivalent to 
```python
out = cubic_to_point(img)
out.save()
```
But `out` is never used anywhere so why not use the first form

Reply:
> I have to admit that cubic_to_point() doesn't really hint that it would return an Image, that's probably the confusing part.

Yeah! By splitting it up, it's clearer that:
* `cubic_to_point()` returns an `Image`
* That `save()` is a method of an `Image` object.

Mostly just subjective / personal preference, though, considering how much back and forth goes on in threads like this: https://stackoverflow.com/questions/1103985/method-chaining-why-is-it-a-good-practice-or-not (Lots of interesting discussion, though!)